source/manual/development-pipeline.html.md

```diff
@@ -54,12 +54,17 @@ Only when the tests pass and the code has been approved the Pull Request can be
 
 In exceptional circumstances, we may wish to block or _freeze_
 deployments for a short period of time. This should be done by
+running a GitHub Action to toggle off continuous deployment and
 checking "Freeze deployments?" and adding an explanatory note
 to the application in the [Release app][release].
 
-> Checking "Freeze deployments?" will turn off all automatic
-> deployments for the application. You can still deploy urgent
-> changes manually if necessary.
+> Code freezes are now handled by toggling off Continuous deployments via a
+> Github action named "Set automatic deploys" for the application.
+> See [here](https://github.com/alphagov/government-frontend/actions/workflows/set-automatic-deploys.yml) for government-frontend.
+>
+> Checking "Freeze deployments?" on Release will turn add an "Automatic deployments disabled"
```

Comment:
```suggestion
> Checking "Freeze deployments?" on Release will add an "Automatic deployments disabled"
```

Reply:
Sorry was juggling a couple of things. Thanks James!